docs/UserGuide.md

```diff
@@ -428,6 +494,10 @@ Appointment on 18-11-2024 18:00 with Patient John Doe, S1234567A.
 Appointment on 20-11-2024 18:00 with Patient Hela, S9876543A.
 
 ```
+---
+## Saving/Retrieving Records and AppointmentRecord Data
```

Comment:
Perhaps can change this to Saving/Retrieving Patient Records and Appointment Records Data (plural form for Records)
(listing down the two types of Records for clarity) 
Change the table of contents for this section too

Reply:
Actually in the naming of the files, there is one plural and one singular so I am not very sure about that

Reply:
I think the naming of the files doesn't matter as users can't see it. They can only see the User Guide